src/statusUpdate.js

```diff
@@ -0,0 +1,14 @@
+// This function updates the status (completed: true/false) of a task
+function updateTaskStatus(taskIndex, status, tasksLocal) {
+  tasksLocal[taskIndex - 1].completed = status;
+  localStorage.setItem('tasks', JSON.stringify(tasksLocal));
+}
+
+// This function clears all completed tasks from the list
+function clearCompletedTasks(tasksLocal) {
+  tasksLocal = tasksLocal.filter((task) => !task.completed);
+  localStorage.setItem('tasks', JSON.stringify(tasksLocal));
+  document.location.reload();
+}
```

Comment:
Good job so far 👍🏼  , but you still need to update indexes after removing completed tasks as not doing so is introducing bags when trying to updateTaskStatus 

![Capture d’écran du 2023-07-28 11-38-12](https://github.com/78fatemanazari/To-Do-List/assets/59707859/a83e88d9-9298-4600-89e2-7690ed4eb8d8)

Reply:
Hi dear @souad988 
I don't have any error while editing the tasks.
![Capture](https://github.com/78fatemanazari/To-Do-List/assets/119149574/02b4648e-8802-4000-ac7e-ffb66b377f75)

Reply:
hi @78fatemanazari  , the problem is after you clear completed tasks if the list has [1,2,3,4,5] as item indexes and 1 3 4 are completed tasks then after clearCompletedTaskes the list will be  [2,5]  when we try to check task 2 as completed edit function looks for tasks[index - 1] => tasks[1] while it index 1 doesn't exists so we need to update indexes after clearCompletedTaskes